src/main/java/dev/jbang/catalog/CatalogUtil.java

```diff
@@ -9,10 +9,7 @@
 import java.nio.file.InvalidPathException;
 import java.nio.file.Path;
 import java.nio.file.Paths;
-import java.util.AbstractMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
+import java.util.*;
```

Comment:
Make sure you run `./gradlew spotlessApply` before committing, no `*` imports are allowed.
I always run `./gradlew spotlessApply clean build installDist` to make sure all my code has been cleaned up.